src/native/minipal/asansupport.cpp

```diff
@@ -0,0 +1,185 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I would suggest having only the code that's different for each platform here and have the rest out of the minipal. The intent of the minipal is to contain just Windows / Unix specific functionality separated into those two folders and all the methods being on the VMToOSInterface.

Reply:
This is in the src/native minipal, not the CoreCLR minipal (the src/native minipal is just a set of free functions with a C-like interface)